### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by steveno**
* Remove unused parameter
* Add some missing parens

### CODE REVIEW

**Comment by steveno** [comment]
@masatake I misunderstood what the TODO was referencing. I thought the "warning" was the missing parens.

**Comment by steveno** [comment]
It looks like the build is failing. I'll look into fixing the issues and get back to this. I'll also look into trying to write a test case. Thank you for help/tips!

**Comment by masatake** [comment]
@steveno, please, se my inline comments for your patch. The test failed because you removed a parameter from a function for long regex flag.

File: main/lxcmd.c [hunk 1]
```diff
@@ -826,8 +828,10 @@ static boolean parseExtensionFields (tagEntry *const entry, char *const string,
 
 						}
 					}
-					else
-						; /* TODO: warning */
+					else {
+                                            /*TODO Handle warning*/
+                                            ;
+                                        }
```
Comment (b4n) [inline main/lxcmd.c]: Style doesn't appear consistent here.  Should probably be either left as is (how is it harmful?) or follow common style, something like:
```C
else
{
	/* TODO: Handle warning */
}
```

**Comment by b4n** [comment]
All newly inserted lines should follow the indentation style (e.g. tabs, width=4)

**Comment by steveno** [comment]
@masatake I've made the changes to my patch. 

@b4n I've made the commit message more explicit to answer your question.

**Comment by steveno** [comment]
@masatake I've lost your comment on making the test. Would you be kind enough repost it?

**Comment by masatake** [comment]
@b4n wrote:
> All newly inserted lines should follow the indentation style (e.g. tabs, width=4)

I am afraid that I myself may violate the style.
Could you make an ideal editor setting with
https://www.wireshark.org/tools/modelines.html
?
So people can follow the rule a bit easier.

**Comment by masatake** [comment]
@steveno, see #607.

**Pull request merged**